docs/guides/elasticsearch/x-pack/x-pack-monitoring.md

```diff
@@ -0,0 +1,539 @@
+---
+title: X-Pack Monitoring of Elasticsearch Cluster in KubeDB
+menu:
+  docs_0.8.0:
+    identifier: es-x-pack-monitoring
+    name: X-Pack Monitoring
+    parent: es-x-pack
+    weight: 20
+menu_name: docs_0.8.0
+section_menu_id: guides
+---
+> New to KubeDB? Please start [here](/docs/concepts/README.md).
+
+# X-Pack Monitoring with KubeDB Elasticsearch
+
+This tutorial will show you how to use X-Pack monitoring in an Elasticsearch cluster in KubeDB.
+
+## Before You Begin
+
+At first, you need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [minikube](https://github.com/kubernetes/minikube).
+
+Now, install KubeDB cli on your workstation and KubeDB operator in your cluster following the steps [here](/docs/setup/install.md).
+
+As KubeDB uses [Search Guard](https://search-guard.com/) plugin for authentication and authorization, you have to know how to configure Search Guard for both Elasticsearch cluster and Kibana. If you don't know, please visit [here](https://docs.search-guard.com/latest/main-concepts).
+
+To keep things isolated, this tutorial uses a separate namespace called `demo` throughout this tutorial.
+
+```console
+$ kubectl create ns demo
+namespace "demo" created
+
+$ kubectl get ns demo
+NAME    STATUS  AGE
+demo    Active  5s
+```
+
+> Note: Yaml files used in this tutorial are stored in [docs/examples/elasticsearch](https://github.com/kubedb/cli/tree/master/docs/examples/elasticsearch) folder in github repository [kubedb/cli](https://github.com/kubedb/cli).
+
+## Overview
+
+At first, we will create some necessary Search Guard configuration and roles to give an user permission to monitor an Elasticsearch cluster from Kibana. We will create a secret with this configuration files. Then we will provide this secret in `spec.databaseSecret` field of Elasticsearch crd so that our Elasticsearch cluster start with this configuration. We will also configure Elasticsearch cluster with a [custom configuration](/docs/guides/elasticsearch/custom-config/overview.md) file.
+
```

Comment:
`We will also configure Elasticsearch cluster with a [custom configuration](/docs/guides/elasticsearch/custom-config/overview.md) file.`

Change to

`We are going to configure Elasticsearch cluster to send xpack monitoring data to an external Elasticsearch cluster using a [custom configuration](/docs/guides/elasticsearch/custom-config/overview.md) file.`

Reply:
changed